src/codecs/jpeg/encoder.rs

```diff
@@ -765,7 +855,8 @@ fn build_quantization_segment(m: &mut Vec<u8>, precision: u8, identifier: u8, qt
     }
 }
 
-fn encode_coefficient(coefficient: i32) -> (u8, u16) {
+#[cfg(feature = "benchmarks")]
+fn encode_coefficient_old(coefficient: i32) -> (u8, u16) {
```

Comment:
I don't think we should keep around an old implementation simply to benchmark against